generators/Exercises.fs

```diff
@@ -0,0 +1,448 @@
+        match key with 
+        | "expected" -> 
+            match canonicalDataCase.Properties.["target"] :?> int64 with
+            | 0L -> Some "int list option"
+            | _  -> None
+        | _ -> None
+
+type CryptoSquare() =
+    inherit Exercise()
+
+type DifferenceOfSquares() =
+    inherit Exercise()
+
+type Gigasecond() =
+    inherit Exercise()
+
+    override this.RenderExpected (canonicalDataCase, key, value) = value :?> DateTime |> formatDateTime |> parenthesize
+ 
+    override this.RenderInput (canonicalDataCase, key, value) =
+        match key with
+        | "input" -> DateTime.Parse(string value, CultureInfo.InvariantCulture) |> formatDateTime |> parenthesize
+        | _ -> base.RenderInput (canonicalDataCase, key, value)
+
+    override this.AdditionalNamespaces = [typeof<DateTime>.Namespace]
+
+type HelloWorld() =
+    inherit Exercise()
+
+type Isogram() =
+    inherit Exercise()
+
+type KindergartenGarden() =
+    inherit Exercise()
+
+    let toPlant (jToken: JToken) =  sprintf "Plant.%s" (jToken.ToString() |> String.humanize)
+
+    override this.RenderExpected (canonicalDataCase, key, value) = 
+        value :?> JArray 
+        |> Seq.map toPlant
+        |> formatList
+
+    override this.RenderSutProperty canonicalDataCase =
+        match Map.containsKey "students" canonicalDataCase.Properties with
+        | true  -> "plantsForCustomStudents"
+        | false -> "plantsForDefaultStudents"
+
+    override this.PropertiesWithIdentifier canonicalDataCase = ["student"; "students"; "diagram"; "expected"]
+
+    override this.UseFullMethodName canonicalDataCase = true
+
+type Leap() =
+    inherit Exercise()
+
+type Luhn() =
+    inherit Exercise()
+
+type Minesweeper() =
+    inherit Exercise()
+
+    override this.RenderValueWithoutIdentifier (canonicalDataCase, key, value) =        
+        value :?> JArray
+        |> normalizeJArray
+        |> Seq.map formatValue
+        |> formatMultiLineList
+
+    override this.PropertiesWithIdentifier canonicalDataCase = ["input"; "expected"]
+
+    override this.IdentifierTypeAnnotation (canonicalDataCase, key, value) = 
+        match value :?> JArray |> Seq.isEmpty with 
+        | true  -> Some "string list"
+        | false -> None
+
+type Pangram() =
+    inherit Exercise()
+
+type PigLatin() =
+    inherit Exercise()
+
+type QueenAttack() =
+    inherit Exercise()
+
+    override this.MapCanonicalDataCaseProperty (canonicalDataCase, key, value) =
+        match canonicalDataCase.Property, key, value with
+        | "create", "expected", _ -> value :?> int64 <> -1L |> box
+        | _ -> base.MapCanonicalDataCaseProperty (canonicalDataCase, key, value)
+
+    override this.RenderInput (canonicalDataCase, key, value) =
+        let parsePositionTuple (tupleValue: obj) =
+            let parts = 
+                (tupleValue :?> JToken)
+                    .SelectToken("position")
+                    .ToObject<string>()
+                    .TrimStart('(')
```

Comment:
`Trim('(')` should trim both the start and end

Reply:
Yes, but I don't want that. I want to trim the start '(' and trim the end ')'.